tests/build_env_test.py

```diff
@@ -167,4 +168,17 @@ def test_build_env_docker_build(mocker):
 
     mocker.patch('docker_build.build_with_docker', return_value=0)
 
+    mocker.patch('pkg_resources.get_distribution', return_value=None)
+
     assert build_env.build_env(arg_strings) == 0
+
+def test_build_env_if_no_conda_build(mocker):
```

Comment:
I think @bethune-bryant was asking for another test similar to this that:
1. Still patched pkg_resources.get_distribution to throw the `pkg_resources.DistributionNotFound` exception.
2. Patched `docker_build.build_with_docker` to return 0.
3. Passed in the "--docker_build" option as one of the values to `build_env`.

Reply:
@bnemanich 
That's exactly what I was thinking. So we can make sure that even if conda-build doesn't exist the script can still do a docker build.
We'll still want to keep the test that you have here now as well, because we want to make sure that case fails.

Reply:
I also have the test case which you have mentioned here https://github.com/open-ce/open-ce/pull/65/files/e850576c4c10f0eac7a8c07e487f494eb159b2f4#diff-0ac0b7c73e283818f092b25e37520c43R171. To cover this scenario, I've modified the existing test case and patched pkg_resources.get_distribution to return None (instead of throwing an exception because get_distribution returns None if the package is not found and then throw the exception). So, effectively the test case ensures that even if conda-build is not present the docker_build.build_with_docker succeeds.